### ISSUE TITLE
[BUG] github :  LGTM C/C++analysis fails

### ISSUE THREAD

**Issue by fgast** [OpenSIPS/opensips#2194]
**Describe the bug**
After creating a pull request which contains "C/C++ -code" changes for the opensips repo, the LGTM analysis fails

**To Reproduce**

1. Create pull request to opensips github repo with change in  C sources
2. Check status of checks.
![Screenshot 2020-08-03 at 10 30 36](https://user-images.githubusercontent.com/40628118/89162520-7c816980-d574-11ea-8473-a0ce49802394.png)

**Expected behavior**
All checks should pass. 

**Relevant System Logs**
Example: 
https://github.com/OpenSIPS/opensips/pull/2193/checks?check_run_id=939584662

Logs: 
https://lgtm.com/projects/g/OpenSIPS/opensips/logs/rev/pr-375235743ddf12ad1bc3099b4621f161ecbc48a8/lang:cpp/stage:Build merge_5af95b6ca670d2915618bb4bc1b138ec7e7da4f1

```[2020-08-03 07:58:18] [build]     index:
[2020-08-03 07:58:18] [build]       before_index:
[2020-08-03 07:58:18] [build]         - export FASTER=1
[2020-08-03 07:58:18] [build]         - export NICER=0
[2020-08-03 07:58:18] [build]       build_command: make exclude_modules="db_oracle osp sngtc cachedb_cassandra cachedb_couchbase cachedb_mongodb auth_jwt" all
[2020-08-03 07:58:18] [build] Failed to parse file: /opt/out/lgtm.effective.yml: Unrecognized Property: "before_index", allowed properties are: "dotnet", "vstools_version", "msbuild", "solution", "build_command", "script", "all_solutions" and "nuget_restore" on line 35 col 9
[2020-08-03 07:58:18] [build] (eventual cause: UnrecognizedPropertyException "Unrecognized field "before_index" (class com.semmle.cloud.model.project_config.L...")
```

**Comment by lemenkov** [comment]
I was just hit by this (#2212)

**Comment by razvancrainea** [comment]
Thanks for the report! I've just pushed a fix, will keep an eye on it to see if it fixes the problem.

**Comment by liviuchircu** [comment]
Actually, one more fix was needed here (c4c7a3e975), since LGTM's build environment is Ubuntu-based.  The more recent PRs (e.g. #2233) seem to be passing, so I think you can close this one, @razvancrainea.

**Comment by lemenkov** [comment]
Can confirm - now it does work!
